aiohttp/web_urldispatcher.py

```diff
@@ -414,6 +414,65 @@ def __repr__(self) -> str:
         return f"<PlainResource {name} {self._path}>"
 
 
+class GroupPlainResource(Resource):
+    def __init__(self) -> None:
+        super().__init__()
+        self._resources: Dict[str, Resource] = {}
+
+    @property
+    def canonical(self) -> str:
+        return "/"
```

Comment:
Is this useful... Are there cases where this might cause problems?

Reply:
fixed

Reply:
lost relevance